apps/pom.xml

```diff
@@ -970,11 +966,6 @@
                             <include>**/*IT.java</include>
                             <include>**/*E2E.java</include>
                         </includes>
-
-                        <!-- Needed to avoid test failures when comparing ETL timestamps. -->
-                        <environmentVariables>
-                            <TZ>UTC</TZ>
-                        </environmentVariables>
```

Comment:
Unsure which approach is better–this has been here for years and only applies to failsafe (integration) tests. The test that _had_ been throwing the most errors is currently called under surefire (unit) testing. Probably a solid argument to be made that it belongs under integration, but still...

Should we provide the UTC timezone via `TZ=UTC` as an envvar or `-Duser.timezone=UTC` as a system property?